documentation/modules/upgrading-mtv-ui.adoc

```diff
@@ -14,13 +14,58 @@ You must upgrade to the next release without skipping a release, for example, fr
 
 .Procedure
 
-. Upgrade the {operator-name}.
+. Change the update channel:
 +
 ifeval::["{build}" == "upstream"]
-See link:https://docs.okd.io/latest/operators/admin/olm-upgrading-operators.html[Upgrading installed Operators] in the {ocp} documentation.
+See link:https://docs.okd.io/latest/operators/admin/olm-upgrading-operators.html#olm-changing-update-channel_olm-upgrading-operators[Changing update channel] in the {ocp} documentation.
 endif::[]
 ifeval::["{build}" == "downstream"]
-See link:https://access.redhat.com/documentation/en-us/openshift_container_platform/{ocp-version}/html-single/operators/index#olm-upgrading-operators[Upgrading installed Operators] in the {ocp} documentation.
+See link:https://access.redhat.com/documentation/en-us/openshift_container_platform/{ocp-version}/html-single/operators/index#olm-changing-update-channel_olm-upgrading-operators[Changing update channel] in the {ocp} documentation.
+endif::[]
+
+. Restart the `CatalogSource` pod:
+
+.. In the the {ocp-short} web console, navigate to the *Operators* -> *Installed Operators* -> *Forklift Operator* -> *Operator details* page. On the *Subscriptions* tab, note the name in the *CatalogSource* field.
+..  From the command line, run the following command:
```

Comment:
This command retrieves a list of pods.

Reply:
Of course it does, because you have not specified the pod name. However, in the first step, you found the catalog source pod name. So why don't you just have the user get the catalog pod instead of every pod in the openshirt-marketplace namespace? That would be much easier and more user friendly.